test/conformance/resources_test.go

```diff
@@ -51,12 +48,19 @@ func TestCustomResourcesLimits(t *testing.T) {
 			corev1.ResourceMemory: resource.MustParse("350Mi"),
 		},
 	}
-	names, err := CreateRouteAndConfig(clients, logger, imagePath, &test.Options{ContainerResources: resources})
+
+	names := test.ResourceNames{
+		Service: test.AppendRandomString("test-resource-limits-", logger),
+		Image:   "bloatingcow",
+	}
+
+	_, err := test.CreateRunLatestServiceReady(logger, clients, &names, &test.Options{ContainerResources: resources})
 	if err != nil {
-		t.Fatalf("Failed to create Route and Configuration: %v", err)
+		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
 	}
-	test.CleanupOnInterrupt(func() { TearDown(clients, names, logger) }, logger)
-	defer TearDown(clients, names, logger)
+
+	test.CleanupOnInterrupt(func() { tearDown(clients, names) }, logger)
+	defer tearDown(clients, names)
 
 	logger.Infof("When the Revision can have traffic routed to it, the Route is marked as Ready.")
```

Comment:
This is probably unnecessary. The CreateRunLatestServiceReady will have already validated that the service went into "Ready" meaning that Config/Route are Ready. It also validates that the domain is populated.

Reply:
Removed.